Third Party Notices.md

```diff
@@ -129,15 +152,29 @@ SOFTWARE.
 
 ## Unity Utilities
 
-- https://github.com/CapnRat/Unity-Utilities
+Component Name: Unity Utilities
 
-The MIT License (MIT)
+License Type: MIT
```

Comment:
are we actually using this anywhere?

Reply:
I think we are for the boolean experimental feature right ?